### ISSUE TITLE
Thread Performance Checker Warning

### ISSUE THREAD

**Issue by nikhiltekwani09** [facebookincubator/SocketRocket#681]
Hi

I m getting following warning
Thread Performance Checker: Thread running at User-interactive quality-of-service class waiting on a lower QoS thread running at Default quality-of-service class. Investigate ways to avoid priority inversions

This error is coming because we are using dispatch_group_wait SRRunLoopThread class. Can some one tell me how to get rid of this warning.

**Comment by funnel20** [comment]
Duplicate of https://github.com/facebookincubator/SocketRocket/issues/648 and https://github.com/facebookincubator/SocketRocket/issues/661
It has been resolved in version 0.7.0.

For a manual solution to your version 0.6.1, see: https://github.com/facebookincubator/SocketRocket/issues/661#issuecomment-1703842082
